pkg/types/clusterconfig/network_validations.go

```diff
@@ -52,7 +55,10 @@ func VerifyNetworkQuotas(
 		_securityGroupRulesQuotaCode,
 	}
 
-	serviceCodes := []string{"ec2", "vpc"}
+	serviceCodes := []string{"vpc"}
+	if !clusterAlreadyExists {
```

Comment:
You recommended splitting the desired quota codes into VPC quota codes and EC2 quota codes. Given this if statement, I think it makes sense to split it out now.